### PR TITLE
Handle reloading of content

### PR DESCRIPTION
**Pull request by rudolf-adamkovic**

### CODE REVIEW

**Comment by nasakinmaxim** [comment]
In review.

**Comment by rudolf-adamkovic** [comment]
@nasakinmaxim Merged another one and now we have conflicts. Looking into it.

**Comment by rudolf-adamkovic** [comment]
@nasakinmaxim Conflicts resolved.

File: AudioBar/AudioBar.swift [hunk 1]
```diff
@@ -80,6 +79,21 @@ public struct AudioBar: Elm.Module {
         switch message {
 
         case .prepareToLoad(let url):
+            let isPlayerActive: Bool = {
+                switch model {
+                case .waitingForURL:
+                    return false
+                case .readyToLoadURL:
```
Comment (nasakinmaxim) [inline AudioBar/AudioBar.swift:75]: Maybe we should have `.waitingForURL` and `.readyToLoadURL` in one line.

File: AudioBar/AudioBar.swift [hunk 2]
```diff
@@ -80,6 +79,21 @@ public struct AudioBar: Elm.Module {
...
+                    return false
+                case .waitingForPlayerToBecomeReadyToPlayURL:
+                    return true
+                case .readyToPlay:
```
Comment (nasakinmaxim) [inline AudioBar/AudioBar.swift:79]: And `. waitingForPlayerToBecomeReadyToPlayURL ` and `. readyToPlay` as well.

File: AudioBar/AudioBar.swift (hunk 1 above)
Comment (rudolf-adamkovic) [reply AudioBar/AudioBar.swift:75]: I tried it and it was less readable.

File: AudioBar/AudioBar.swift (hunk 2 above)
Comment (rudolf-adamkovic) [reply AudioBar/AudioBar.swift:79]: I tried it and it was less readable.

File: AudioBar/AudioBar.swift (hunk 1 above)
Comment (nasakinmaxim) [reply AudioBar/AudioBar.swift:75]: Understood.

File: AudioBar/AudioBar.swift (hunk 2 above)
Comment (nasakinmaxim) [reply AudioBar/AudioBar.swift:79]: Understood.

**Review by nasakinmaxim — APPROVED** [review]
LGTM.

**Pull request merged**